open_rarity/resolver/testset_resolver.py

```diff
@@ -491,13 +508,17 @@ def _rank_diff(rank1: int | None, rank2: int | None) -> int | None:
 def serialize_to_csv(
     collection_with_metadata: CollectionWithMetadata,
     tokens_with_rarity: list[TokenWithRarityData],
-) -> None:
+    dry_run: bool = False,
+) -> list | None:
     """Serialize collection and ranking data to CSV
 
     Parameters
     ----------
-    collection : Collection
-        collection
+    collection_with_metadata : Collection
+    dry_run: bool
+        If set to True, the CSV will not be written to disk but returned as an
+        array of rows
+
```

Comment:
`dry_run` doesn't quite feel right here... Maybe something like `to_console` and it still formats that csv but just prints to stdout?